backend/sprocs.py

```diff
@@ -34,8 +34,9 @@ def get_all_job_postings(db_filename) -> str:
                 company_name,
                 job_title,
                 description,
+                preprocessed_description,
                 salary,
-                STRING_AGG(t.name, ', ') AS tags,
+                group_concat(t.name, ', ') AS tags,
```

Comment:
I like to keep method calls highly visible.  Can you cast to uppercase() on `GROUP_CONCAT`.

Reply:
GROUP_CONCAT uppercase() complete.